rpcs3/rpcs3qt/curl_handle.cpp

```diff
@@ -66,3 +65,11 @@ std::string curl_handle::get_verbose_error(CURLcode code)
 }
 
 }
+
+#ifdef _WIN32
+// Function exported from our user_settings.h in WolfSSL, implemented in RPCS3
+extern "C" FILE* wolfSSL_fopen_utf8(const char* name, const char* mode)
+{
+	return _wfopen(utf8_to_wchar(name).c_str(), utf8_to_wchar(mode).c_str());
+}
+#endif
```

Comment:
Need empty new line at the end of the file(see https://github.com/RPCS3/rpcs3/wiki/Coding-Style ).